### PR TITLE
enable test/nn/test_pooling.py

### PR DESCRIPTION
**Pull request by kshitij12345**
Ref: https://github.com/pytorch/pytorch/pull/84165#discussion_r956693153

Ref: https://github.com/pytorch/pytorch/pull/83915

### CODE REVIEW

**Comment by kshitij12345** [comment]
cc: @JackCaoG

**Comment by kshitij12345** [comment]
Gentle ping @JackCaoG

File: test/pytorch_test_base.py [hunk 1]
```diff
@@ -275,6 +275,11 @@
         'test_Dropout2d_xla',  # Started to pass
     },
 
+    # test/nn/test_pooling.py
+    'TestPoolingNNDeviceTypeXLA': {
+        'test_pool_invalid_size',  # expecting a different runtime error
```
Comment (JackCaoG) [inline test/pytorch_test_base.py:280]: can you remove the same entry in `test_nn` above?

Comment (kshitij12345) [reply test/pytorch_test_base.py:280]: Ah. Right. (And sorry I forgot yesterday was a holiday 😅)

**Review by JackCaoG — APPROVED** [review]

**Comment by JackCaoG** [comment]
You can merge the pytorch one, I will merge this one right after.(assuming pytorch pr don't need this pr to be merged first which I think is the case)

**Comment by JackCaoG** [comment]
actually I take it back, we need to merge this pr first. Otherwise disabled test will get run.

**Comment by JackCaoG** [comment]
This is slightly annoying since this pr is at your separate fork, so what we have to do is 
1. merge this pr
2. take the new master hash and update the https://github.com/pytorch/pytorch/blob/master/.github/ci_commit_pins/xla.txt
3. force merge your pytorch pr

I can help you with the process tmr.

**Comment by kshitij12345** [comment]
Actually, the Pytorch PR has actually been merged before. It didn't cause any failures back then as the new tests were not run at all.

So we just have to merge this PR

**Pull request merged**